packages/bruno-app/src/components/ResponsePane/index.js

```diff
@@ -116,8 +116,13 @@ const ResponsePane = ({ rightPaneWidth, item, collection }) => {
         ) : null}
       </div>
       <section className={`flex flex-grow ${focusedTab.responsePaneTab === 'response' ? '' : 'mt-4'}`}>
-        {isLoading ? <Overlay item={item} collection={collection} /> : null}
-        {getTabPanel(focusedTab.responsePaneTab)}
+        {isLoading && <Overlay item={item} collection={collection} />}
+
+        {response.isError ? (
+          <span className="mt-5 px-5 text-red-500">{response.error}</span>
+        ) : (
+          getTabPanel(focusedTab.responsePaneTab)
```

Comment:
Instead of always displaying the error, Can we pass this error to the `QueryResult` component and display it there.

```html
<QueryResult
  item={item}
  collection={collection}
  width={rightPaneWidth}
  data={response.data}
  error={response.error}
  headers={response.headers}
/>
```

That way, the user can switch to the timeline tab and see the request that was sent.